src/App.tsx

```diff
@@ -1,51 +1,40 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { NavLink, Outlet } from 'react-router-dom';
+import classNames from 'classnames';
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
       className="navbar is-light is-fixed-top is-mobile has-shadow"
       data-cy="Nav"
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              'navbar-item', { 'is-active': isActive },
+            )}
+          >
+            Home
+          </NavLink>
+          <NavLink
+            to="/tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item', { 'is-active': isActive },
+            )}
+          >
+            Tabs
+          </NavLink>
```

Comment:
Put nav to a Navigation component and create a Layout component that will serve as a wrapper for all your routes. Within this Layout, we'll include a Navigation component and a section to wrap the content. To achieve this, we can use the Outlet component from react-router-dom.

```
<Routes>
  <Route path="/" element={<Layout />}>
    .. homepage
    ...tabs
    ...
 </Routes>   
```